content/docs/contribute/shortcodes/superscripts.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: codelab
+title: Superscripts
+description: Superscripts with links
+draft: false
+order: 6
+---
+
+{{<br>}}
+
+### Superscripts (sup)
+
+Superscript go next to text.
+
+#### Example
+
+Regular Text{{< sup >}}Superscript{{</ sup >}}
+
+```go-html-template
+  Regular Text{{</* sup */>}}Superscript{{</* / sup */>}}
+```
+
+Atsign{{< sup >}}{{< a href="https://atsign.com" target="_blank" rel="calonical" >}}1{{</ a>}}{{</ sup >}}
+
+```go-html-template
+  Atsign{{</* sup */>}}{{</* a href="https://atsign.com" target="_blank" rel="calonical" */>}}1{{</*/ a */>}}{{</*/ sup */>}}
```

Comment:
No need for this second example, and also it is canonical not calonical.

Reply:
See 6a5eac242114688272200d9c1f926c74f3f5e48a